app.js

```diff
@@ -49,6 +46,6 @@ function handler (req, res) {
 
 io.on('connection', function (socket) {
   socket.on('user_favorite_streamers', function (data) {
-    start_iteration(data);
+    streamers = data;
```

Comment:
you are a heartbeat user, and i am a heartbeat user.

you follow streamers : x, y, z
i follow streamers: a, b, c

you connect to the socket, therefore streamers = [x, y, z].
after 15 mins, socket broadcasts
you receive broadcast
i connect to the socket, therefore streamers = [a, b, c]
after 15 mins, socket broadcasts
i receive status of a,b,c
you receive status of a,b,c

what happened to x,y,z?